subsys/bluetooth/audio/has_client.c

```diff
@@ -22,6 +24,7 @@
 
 enum {
 	HAS_DISCOVER_IN_PROGRESS,
+	HAS_CP_OPERATION_IN_PROGRESS,
```

Comment:
Can we reuse the same flag? 

I guess that regardless of a discover or a CP operation is in progress, we shouldn't allow any other operation?

Reply:
Not exactly, as I use `HAS_DISCOVER_IN_PROGRESS` in one, additional specific check